manager/logbroker/broker.go

```diff
@@ -383,6 +383,15 @@ func (lb *LogBroker) PublishLogs(stream api.LogBroker_PublishLogsServer) (err er
 			}
 		}
 
+		// if we have a close message, close out the subscription
+		if log.Close {
+			// Mark done and then set to nil so if we error after this point,
```

Comment:
I think we are always setting an error? The `err` value comes from the return value of the function.